packages/dashboard/src/components/appbar.tsx

```diff
@@ -40,51 +42,129 @@ export default function AppBar(props: AppBarProps): React.ReactElement {
 
   return (
     <MuiAppBar id="appbar" position="static">
-      <Toolbar>
-        <Typography variant="h6" className={classes.toolbarTitle}>
-          Dashboard
-        </Typography>
+      {tooltips && (
+        <Toolbar>
+          <Typography variant="h6" className={classes.toolbarTitle}>
+            Dashboard
+          </Typography>
 
-        <IconButton id="toggle-omnipanel-btn" color="inherit" onClick={() => toggleShowOmniPanel()}>
-          <DashboardIcon />
-        </IconButton>
-
-        <IconButton id="show-settings-btn" color="inherit" onClick={() => showSettings(true)}>
-          <SettingsIcon />
-        </IconButton>
-        {user && (
-          <>
+          <Tooltip
```

Comment:
I think we can improve this solution, maybe we can create our own Tooltip component that receives the tooltip state and decides if It'll return just the child or the child with the tooltip wrapped. E.g:
```
<CustomTooltip tooltip=true>
    <ChildComponent>
</CustonTooltip>
```

returns:

```
<Tooltip>
    <ChildComponent>
</Tooltip>
```


```
<CustomTooltip tooltip=false>
    <ChildComponent>
</CustonTooltip>
```

returns:

```

    <ChildComponent>

```

Reply:
Ok let me work on this!